security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-operator-integrating-istio"]
+= Integrating the {cert-manager-operator} with Istio-CSR
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-integrating-istio
+
+toc::[]
+
+:FeatureName: Istio-CSR integration for {cert-manager-operator}
+include::snippets/technology-preview.adoc[]
+
+The {cert-manager-operator} provides enhanced support for securing workloads and control plane components in {SMProductName} or Istio. This includes support for certificates enabling mutual TLS (mTLS), which are signed, delivered, and renewed using cert-manager issuers. You can secure Istio workloads and control plane components by using the {cert-manager-operator} managed Istio-CSR agent.
+
+With this Istio-CSR integration, Istio can now obtain certificates from the {cert-manager-operator}, simplifying security and certificate management.
+
+[id="cert-manager-operator-istio-csr-installing"]
+== Installing the Istio-CSR agent through cert-manager Operator for Red Hat OpenShift
+
+// Enabling Istio-CSR
+include::modules/cert-manager-enabling-istio.adoc[leveloffset=+2]
+
+// Creating issuer
+include::modules/cert-manager-istio-creating-issuer.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="cert-manager-operator-istio_additional-resources"]
+=== Additional resources
```

Comment:
Since this additional resources section applies only to a module and not to the entire assembly, it needs to be formatted differently.
```suggestion
[role="_additional-resources"]
.Additional resources
```

See the [docs guidelines](https://github.com/openshift/openshift-docs/pull/90252) for more info.

Reply:
And still address this issue